src/test/java/core/basesyntax/serviceimpl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.serviceimpl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceImplTest {
+    private static final String HEADERS = "fruit,quantity" + System.lineSeparator();
+
+    @BeforeAll
+    static void clear() {
+        Storage.storage.clear();
+    }
+
+    @AfterEach
+    void clearAfter() {
+        Storage.storage.clear();
+    }
```

Comment:
can be replaced with one @BeforeEach that clears storage here and in the rest test classes that are using storage